xonsh/environ.py

```diff
@@ -13,7 +13,6 @@
 import string
 import subprocess
 import shutil
-from itertools import chain
```

Comment:
Because it is already imported at line 6 [here](https://github.com/rahiel/xonsh/blob/5f4874e793e4df51495791d11ed7817f9bfb1da5/xonsh/environ.py#L6).